hollow/src/main/java/com/netflix/hollow/api/consumer/HollowConsumer.java

```diff
@@ -462,6 +464,11 @@ public HollowConsumerMetrics getMetrics() {
          * @return the blob of the reverse delta
          */
         HollowConsumer.Blob retrieveReverseDeltaBlob(long currentVersion);
+
+        default Set<String> configuredOptionalBlobParts() {
```

Comment:
nit: alternatively,
`default Optional<Set<String>> configuredBlobParts`
But given the number of touch points maybe not worth it

Reply:
also, came across https://justamonad.com/wrapping-collection-in-optional-is-a-big-no-no/